### ISSUE TITLE
Code component is not accessible when scrolled

### ISSUE THREAD

**Issue by thuey** [dequelabs/cauldron#932]
When the `Code` component scrolls because it does not have keyboard access. Example issue: https://axe.deque.com/issues/a01c8247-0945-4313-afc4-07e04ec38b3e

The `Code` component should wrap lines instead of scrolling by default.

It would also be nice if you could opt-in to scrolling, and when it needs to scroll that it would turn into a focusable region. This could maybe be implemented by using a `ResizeObserver`. However, the `Code` component would need to be wrapped in something else that scrolls, since `react-syntax-highlighter` does not forward refs: https://github.com/react-syntax-highlighter/react-syntax-highlighter/issues/335.
